Source/Scene/PointCloud3DTileContent.js

```diff
@@ -844,7 +844,7 @@ define([
         var hasColorStyle = defined(colorStyleFunction);
         var hasShowStyle = defined(showStyleFunction);
         var hasPointSizeStyle = defined(pointSizeStyleFunction);
-        var hasClippedContent = defined(clippingPlanes) && clippingPlanes.enabled && content._tile._isClipped;
+        var hasClippedContent = defined(clippingPlanes) && clippingPlanes.enabled && content._tile._isClipped && !FeatureDetection.isInternetExplorer();
```

Comment:
Maybe this check could be centralized somewhere? `ClippingPlaneCollection.isSupported`?

For reference: https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Scene/ClassificationPrimitive.js#L340-L348